ztp/source-crs/optional-extra-manifest/enable-crun-master.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
Does everything under the source-crs directory get included into the generated container automatically? @nishant-parekh

Reply:
yes that is correct:
https://github.com/openshift-kni/cnf-features-deploy/blob/c827f233f55e5c4ab5e3c42c907c6783a3ce00cf/ztp/resource-generator/Containerfile#L32